src/metabase/api/collection.clj

```diff
@@ -470,7 +470,7 @@
       true)))
 
 (defn- post-process-card-row [row]
-  (-> row
+  (-> (t2/hydrate (t2/instance :model/Card row) :can_write)
```

Comment:
Total nit, feel free to ignore: but you could thread through `(t2/hydrate :can_write)`.

Reply:
Good idea, done!